integration/shop/package.json

```diff
@@ -7,6 +7,7 @@
     "build": "ng build",
     "build:single-spa": "ng build --prod --deploy-url http://localhost:4200/",
     "serve:single-spa": "ng serve --disable-host-check --port 4200 --deploy-url http://localhost:4200/ --live-reload false",
+    "preinstall": "cp -r ../../src/browser-lib ./lib",
```

Comment:
This is temporary !!
I'm explaining: `${root}/src/browser-lib/single-spa-angular.ts` imports everything from `${root}/node_modules` but we want it to import everything from `integration/shop/node_modules`.
Will be removed in next PR with ng-packagr...